salt/kubelet/stop.sls

```diff
@@ -13,6 +15,10 @@ drain-kubelet:
       - /bin/true
     - require:
       - file: {{ pillar['paths']['kubeconfig'] }}
+  grains.present:
+    - name: kubelet:should_uncordon
+    - value: {{ should_uncordon }}
```

Comment:
why not just executing the statement here instead of caching it in a variable, afaics it's just used once

Reply:
It's basically the same thing, but thought it would be a bit more readable. However I can inline it if you think it's better.